### DISCUSSION TITLE
Vdsql feedback

### DISCUSSION THREAD

**Discussion by frosencrantz** [saulpw/visidata#1579] [Q&A]
Congratulations @saulpw @anjakefala @cpcloud on the new vdql release.

Since discussions are not setup on https://github.com/visidata/vdsql, I take it they should happen here.
One big question I have is what library is converting between python and SQL functions?  That is brilliant!

I did try using `vdsql` but had a couple issues.

1. Installation caused me issues.  `pip install` got into some infinite loop that took a while to resolve.  This might just be my environment, but it did slow me down for a while.  I ended up changing the `requirements.txt` file to
   ```
   visidata >= 2.10.2
   ibis-framework[sqlite] >= 3.2
   ibis-substrait
   sqlparse
   ```
2.  There is an issue with the `z"` command it will convert the generated SQL from `SELECT t0.a FROM mytable AS t0` to `SELECT t0.a FROM (SELECT t1.a FROM my_table AS t1) AS t0`
3. Also, when adding a new column like `a.lower()` and then trying to hide `a`, and look at the side panel it says `name 'a' is not defined` and there is stack trace.

**Comment by anjakefala** [comment]
Hey @frosencrantz! Can you open these as issues in the `vdsql` repo? I forget about bug reports in the Discussions tab really easily! ^^;

Thanks for the continued loving attention to bug hunting. =)

**Comment by frosencrantz** [comment]
Three issues generated in vdsql repo.
